SourceryFramework/Sources/Parsing/Composer.swift

```diff
@@ -78,15 +78,12 @@ public struct Composer {
         let resolutionStart = currentTimestamp()
 
         let resolveType = { (typeName: TypeName, containingType: Type?) -> Type? in
-            return self.resolveType(typeName: typeName, containingType: containingType, unique: unique, modules: modules, typealiases: typealiases)
+            return self.resolveType(typeName: typeName, containingType: containingType, unique: unique, modules: modules, typealiases: resolvedTypealiases)
         }
 
-        let array = Array(unique.values)
-        DispatchQueue.concurrentPerform(iterations: array.count) { (counter) in
-            let type = array[counter]
-            type.typealiases.forEach { (_, alias) in
-                alias.type = resolveType(alias.typeName, type)
-            }
```

Comment:
This is no longer necessary as it's now handled by https://github.com/krzysztofzablocki/Sourcery/pull/814/files#diff-3b9e6e3e3ed536fd38045c732a09f562R111-R115